content/en/docs/languages/net/exporters.md

```diff
@@ -125,112 +151,107 @@ AppContext.SetSwitch("System.Net.Http.SocketsHttpHandler.Http2UnencryptedSupport
+```sh
+dotnet add package OpenTelemetry.Exporter.Console
 ```
 
-Next, install the Zipkin exporter package:
+If you're using ASP.NET Core install the
+[`OpenTelemetry.Extensions.Hosting`](https://www.nuget.org/packages/OpenTelemetry.Extensions.Hosting)
+package as well:
 
-```shell
-dotnet add package OpenTelemetry.Exporter.Zipkin
+```sh
 dotnet add package OpenTelemetry.Extensions.Hosting
 ```
 
-If you're using ASP.NET Core, configure the exporter in your ASP.NET Core
-services:
+### Usage {#console-usage}
+
+#### ASP.NET Core {#console-usage-asp-net-core}
+
+Configure the exporter in your ASP.NET Core services:
 
 ```csharp
 var builder = WebApplication.CreateBuilder(args);
 
 builder.Services.AddOpenTelemetry()
-    .WithTracing(b =>
+    .WithTracing(tracing =>
     {
-        b.AddZipkinExporter(o =>
-        {
-            o.Endpoint = new Uri("your-zipkin-uri-here");
-        })
+        tracing.AddConsoleExporter()
+        // The rest of your setup code goes here too
+    })
+    .WithMetrics(metrics =>
+        metrics.AddConsoleExporter()
         // The rest of your setup code goes here too
     });
+
+builder.Logging.AddOpenTelemetry(logging => {
+        logging.AddConsoleExporter()
+  });
 ```
 
-Otherwise, configure the exporter when creating a tracer provider:
+#### Non-ASP.NET Core {#console-usage-non-asp-net-core}
+
+Configure the exporter when creating a tracer provider:
 
 ```csharp
 using var tracerProvider = Sdk.CreateTracerProviderBuilder()
-    .AddZipkinExporter(o =>
-    {
-        o.Endpoint = new Uri("your-zipkin-uri-here");
-    })
+    .AddConsoleExporter()
 
     // Other setup code, like setting a resource goes here too
 
     .Build();
-```
 
-## Prometheus (Experimental)
+using var meterProvider = Sdk.CreateMeterProviderBuilder()
+    .AddConsoleExporter()
+
+    // Other setup code, like setting a resource goes here too
 
-**\*Note:** this is experimental and dependent on the OpenTelemetry
-specification to be made stable before it will be a released package. For now,
-we recommend using the OTLP exporter and using the OpenTelemetry Collector to
-send metrics to Prometheus\*
+    .Build();
 
-If you're using Prometheus to visualize metrics data, you'll need to set it up
-first. Here's how to do it using a docker container:
+using var loggerFactory = LoggerFactory.Create(builder =>
+{
+    builder.AddOpenTelemetry(logging =>
+    {
+        logging.AddConsoleExporter()
+    });
+});
+```
 
-First, you'll need a `prometheus.yml` file to configure your Prometheus backend,
-such as the following:
+{{% docs/languages/exporters/jaeger %}}
 
-```yml
-global:
-  scrape_interval: 1s
-  evaluation_interval: 1s
+{{% docs/languages/exporters/prometheus-setup %}}
 
-scrape_configs:
-  - job_name: prometheus
-    static_configs:
-      - targets: [localhost:9090]
-```
+### Dependencies {#prometheus-dependencies}
 
-Next, run the following docker command to set up Prometheus:
+Install the
```

Comment:
move this also under asp.net core section.

Reply:
I want to keep the existing pattern of "Dependencies" and "Usage", it's a little bit odd for .NET now, but this is for consistency with other languages, an alternative would be to split it out in core vs non-core first and then go into dependencies and usage, but that might be a separate PR

Reply:
going to merge this PR now we can have incremental changes to improve